lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.spec.ts

```diff
@@ -233,6 +233,38 @@ describe('CardViewTextItemComponent', () => {
             renderChipsForMultiValuedProperties(cardViewTextItemObject, true, 3, '1.1', '2.2', '3.3');
         });
 
+        it('should only render new chip when provided value is valid for specified validators set', async () => {
+            const cardViewTextItemObject = {
+                label: 'Text label',
+                value: [1.1, 2.2, 3.3],
+                key: 'textkey',
+                editable: true,
+                multivalued: true,
+                type: 'float'
+            };
+            component.editable = true;
+            renderChipsForMultiValuedProperties(cardViewTextItemObject, true, 3, '1.1', '2.2', '3.3');
+            const floatValidator: CardViewItemValidator = new CardViewItemFloatValidator();
+            component.property.validators = [floatValidator];
+            const inputElement = fixture.debugElement.query(By.css('[data-automation-id="card-textitem-editchipinput-textkey"]')).nativeElement;
+            component.addValueToList({ value: 'abcd', chipInput: inputElement, input: inputElement });
+            fixture.detectChanges();
+
+            let error: HTMLLIElement = fixture.debugElement.query(By.css('[data-automation-id="card-textitem-error-textkey"] li')).nativeElement;
```

Comment:
this text is used multiple times, maybe we can move it to some const and reuse? 

[data-automation-id="card-textitem-error-textkey"]

Reply:
Good point, moved to a separate function